### PR TITLE
Input

### PR DESCRIPTION
**Pull request by bossmonkey98**
Input component updated

### CODE REVIEW

**Review by sanjitsarkar — APPROVED** [review]
Code looks clean and easy to understand
Good to merge.

File: pages/input/input.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    width:98%;
+    height: 30px;
+    border-radius: 5px;
+    margin: 5px;
+}
+
+.input{
+    margin: 2px;
+    padding: 2px;
+    border: 3px solid lightgreen;
```
Comment (arjun-computer-geek) [inline pages/input/input.css:52]: Always use colour variables bro

File: pages/input/input.css [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    display: flex;
+    flex-direction: column;
+    margin: 5px;
+    padding: 10px;
+    border: 1px solid var(--bg-color);
+    border-radius: 10px;
+}
+.form>input{
+    width:98%;
+    height: 30px;
```
Comment (arjun-computer-geek) [inline pages/input/input.css:44]: i think thi is too much in px? Right?

File: pages/image/image.css [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+.round-img-container{
+    margin:10px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+.round-img{
+    width: 400px;
```
Comment (arjun-computer-geek) [inline pages/image/image.css:21]: Use rem

**Review by arjun-computer-geek — APPROVED** [review]
Do suggested changes . Else looks good.

File: pages/input/input.css (hunk 2 above)
Comment (bossmonkey98) [reply pages/input/input.css:44]: Yeah

File: pages/image/image.css (hunk 3 above)
Comment (bossmonkey98) [reply pages/image/image.css:21]: okay

**Pull request merged**